xds/build.gradle

```diff
@@ -22,6 +22,9 @@ dependencies {
             project(':grpc-core'),
             project(':grpc-services'),
             project(':grpc-auth')
+    compile (libraries.pgv) {
+        exclude group: 'com.google.protobuf'
```

Comment:
Add a comment saying why it is excluded.

Reply:
Done.